certification/README.md

```diff
@@ -0,0 +1,2 @@
+# certification
+Software Carpentry certifications
```

Comment:
I'm adding the script and the required templates to amy itself, that way nothing needs to be installed separately. Should I add it to the docs?